src/NzbDrone.Core.Test/ParserTests/ParserFixture.cs

```diff
@@ -61,6 +61,7 @@ public void should_remove_accents_from_title()
         [TestCase("World.Movie.Z.2.EXTENDED.2013.German.DL.1080p.BluRay.AVC-XANOR", "World Movie Z 2")]
         [TestCase("G.I.Movie.Movie.2013.THEATRiCAL.COMPLETE.BLURAY-GLiMMER", "G.I. Movie Movie")]
         [TestCase("www.Torrenting.org - Movie.2008.720p.X264-DIMENSION", "Movie")]
+        [TestCase("The.French.Movie.2013.720p.BluRay.x264 - ROUGH[PublicHD]", "The French Movie")]
```

Comment:
Add this case to `should_not_parse_wrong_language_in_title` test as well

Reply:
added and pushed